plugins/vmray/komand_vmray/actions/submit_file/action.py

```diff
@@ -18,21 +21,21 @@ def __init__(self):
     def run(self, params={}):
         file_ = params.get("file", None)
         file_name = file_.get("filename")
-        optional_params = params.get("optional_params")
+        optional_params = params.get("optional_params", {})
         analyzer_mode = params.get("analyzer_mode")
         if analyzer_mode != "default":
             optional_params["analyzer_mode"] = analyzer_mode
 
         try:
             file_bytes = base64.b64decode(file_.get("content"))
-        except:
-            raise Exception("Error decoding base64, contents of the file must be encoded with base64!")
+        except [binascii.Error, ValueError]:
+            raise PluginException(preset=PluginException.Preset.BASE64_DECODE)
 
         mime_types, check_pass = self.connection.api.check_filetype(file_bytes)
         if check_pass:
             self.logger.info(f"File types {mime_types} found for file {file_name} and are supported by VMRay")
             resp = self.connection.api.submit_file(file_name, file_bytes, optional_params)
-            clean_data = komand.helper.clean(resp)
+            clean_data = insightconnect_plugin_runtime.helper.clean(resp)
             return {"results": clean_data}
         else:
             self.logger.error(f"File types, not supported by VMRay: {mime_types}")
```

Comment:
I am wondering if here should be raised Plugin exception due to the VMRay file type. Also please use `Output.RESULTS`